src/graphics/program-lib/chunks/skybox.vert

```diff
@@ -5,15 +5,15 @@ attribute vec3 aPosition;
 uniform mat4 matrix_view;
 #endif
 
-uniform mat4 matrix_projection;
+uniform mat4 matrix_projection_skybox;
```

Comment:
Naming convention means this should be `matrix_projectionSkybox`. It's a weird mix of underscore and camel case. But that's the convention.

Reply:
fixed, thanks